modelscan/settings.py

```diff
@@ -87,6 +91,7 @@
         "MEDIUM": {},
         "LOW": {},
     },
+    "reporting": {"module": "modelscan.reports.ConsoleReport", "settings": {}},
```

Comment:
Add a comment here showing how to use JSON reporting with `modelscan.reports.JSONReport`